README.md

```diff
@@ -127,6 +127,8 @@ something that can be coerced into a `URI` such as a `String`) with three option
 * `:bearer_token`: a `String` bearer token to authorize any HTTP requests, defaults to `nil`.
 * `:headers`: a `Hash` containing custom HTTP headers, defaults to `{}`.
 
+Strip an `oai` namespace prefix by default.
```

Comment:
Is this a relevant detail for the user? I'd argue that it's an internal implementation detail and probably not worth noting.

Reply:
Yeah, sounds fair 👍